featurebyte/models/observation_table.py

```diff
@@ -131,6 +131,7 @@ class ObservationTableModel(MaterializedTableModel):
     purpose: Optional[Purpose] = Field(default=None)
     least_recent_point_in_time: Optional[StrictStr] = Field(default=None)
     entity_column_name_to_count: Optional[Dict[str, int]] = Field(default_factory=dict)
+    primary_entity_ids: Optional[List[PydanticObjectId]] = Field(default_factory=None)
```

Comment:
Would it be cleaner if we change this to `primary_entity_ids: List[PydanticObjectId] = Field(default_factory=list)`?

Reply:
sure!